templates/request_page.hbs

```diff
@@ -101,11 +99,11 @@
             {{checkbox 'mark_as_solved'}}
 
             {{#if request.can_be_marked_as_solved}}
-              <a role="button" class="button-large button-secondary mark-as-solved"
+              <button type="button" class="button-large button-secondary mark-as-solved"
```

Comment:
wait, but this one is a submit button, right?

Reply:
No, we do submission manually in the handler after we set `checked` and `disabled`. I didn't change that logic.

Reply:
gotcha! thanks!